CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,2 @@
-[cygnus-common][SQLBackendImpl] Add configuration option to persist errors in SQL sinks (#1928)
-[cygnus-ngsi-ld] Creation of the new PostGIS sink for persisting NGSI-LD notifications (#1905)
-[cygnus-ngsi][MysqlSink] Quoted Sql field names to avoid naming problems. (#1863)
+[cygnus-ngsi-ld]Creation of the new CKAN sink for persisting NGSI-LD notifications
+[cygnus-common] Add new methods in class CKANBackendImpl for allowing the creation of the data store with ngsi-ld notifications
```

Comment:
```suggestion
[cygnus-common][SQLBackendImpl] Add configuration option to persist errors in SQL sinks (#1928)
[cygnus-ngsi-ld] Creation of the new PostGIS sink for persisting NGSI-LD notifications (#1905)
[cygnus-ngsi][MysqlSink] Quoted Sql field names to avoid naming problems. (#1863)
[cygnus-ngsi-ld]Creation of the new CKAN sink for persisting NGSI-LD notifications
[cygnus-common] Add new methods in class CKANBackendImpl for allowing the creation of the data store with ngsi-ld notifications
```
As existing lines in CHANGES_NEXT_RELEASE shouldn't be removed (just add the ones corresponding to the PR)